### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by fallacy321**
@hhaidar  I have removed the sudo npm start instruction as I am able to run it even without sudo. Maybe you can review it :) ?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,6 +62,11 @@ Editor preferences are available in the [editor config][editorconfig] for easy
 use in common text editors. Read more and download plugins at
 <http://editorconfig.org>.
 
+## Note for ubuntu developers 
+You should use sudo npm start instead of npm start. Similarly for other bash commands 
+```
+sudo npm start
```
Comment (hhaidar) [inline README.md]: You should almost never need to use sudo for this..

**Comment by hhaidar** [comment]
Heyt here @fallacy321, are you having any issues running let's chat without sudo?

**Comment by fallacy321** [comment]
Hi @hhaidar Yes, without sudo, I was not able to update the packages. I see now, npm start works though but for updating sudo was required. I am using Kubuntu 13.04
Regards

**Comment by hhaidar** [comment]
Seems like there are 3-4 features in one PR. Do you mind splitting this up?

Also we can't accept the changes to README as `npm update` is not necessary :(

**Pull request closed without merging**